### PR TITLE
adding the possibility to add a city for admin

### LINKED ISSUE #64: admin should be able to add new city to DB (effort 2)
Add in admin menu 'add city' 
When click show ui with input and button to add new City 

(make sure that you finished api)

### PR DESCRIPTION
**Pull request by ibednorz**
Screenshot 2021-04-01 at 22 02 33
Screenshot 2021-04-01 at 22 03 07
Screenshot 2021-04-01 at 22 03 10

### CODE REVIEW

File: src/components/views/adminView/Cities.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react'
+import { TextField, Input, CircularProgress } from '@material-ui/core'
+import { fetchData } from '../../../utils'
+import CheckCircleIcon from '@material-ui/icons/CheckCircle'
+
+import Grid from '@material-ui/core/Grid'
+import Button from '@material-ui/core/Button'
+
+export const Cities = ({ useStyles }) => {
+  const [city, setCity] = useState([])
```
Comment (pdybowski) [inline src/components/views/adminView/Cities.jsx]: why is this an array?

File: src/components/views/adminView/Cities.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  const [adding, setAdding] = useState(false)
+  const [added, setAdded] = useState(false)
+  const classes = useStyles()
+
+  const handlerCityChange = (e) => {
+    setCity(e.target.value)
+  }
+  const addNext = () => {
```
Comment (pdybowski) [inline src/components/views/adminView/Cities.jsx]: i would recomend to change this function name to something like 'goNext' etc.. 

AddNext kojarzy się bardziej z jakaś akcją, dodaniem czegoś

File: src/components/views/adminView/Cities.jsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    try {
+      setAdding(true)
+      const addedCity = await fetchData(
+        global.API_BASE_URL + 'api/admin/city',
+        'POST',
+        {
+          name: city,
+        }
+      )
+      if (addedCity) {
```
Comment (pdybowski) [inline src/components/views/adminView/Cities.jsx]: if request passes we will always have value, so this if statement is not necessary

**Review by pdybowski — COMMENTED** [review]
some small changes

File: src/components/views/adminView/Cities.jsx (hunk 1 above)
Comment (ibednorz) [reply src/components/views/adminView/Cities.jsx]: i missed it

**Pull request merged**